nvm-exec

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-DIR="$(command cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
```

Comment:
could this be considered a breaking change if someone's relying on the _lack_ of realpath here, wrt symlinks?